core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp

```diff
@@ -48,6 +48,30 @@
             "roleType": "NIFI_NODE",
             "base": true,
             "configs": [
+              {
+                "name": "ssl_enabled",
+                "value": "true"
+              },
+              {
+                "name": "ssl_server_keystore_location",
+                "value": "{{CM_AUTO_TLS}}"
+              },
+              {
+                "name": "ssl_server_keystore_password",
+                "value": "{{CM_AUTO_TLS}}"
+              },
+              {
+                "name": "ssl_server_keystore_keypassword",
+                "value": "{{CM_AUTO_TLS}}"
```

Comment:
sure?

Reply:
This value {{CM_AUTO_TLS}} is not a real value, but a flag for CM to set this property value automatically. These property/value pairs were added/tested for 7.2.8 and 7.2.9 and everything works fine, as expected.